File: typescript/src/main/vsdk/toolkit/common/linealAlgebra/Jacobian.ts
export class Jacobian {
  public A: number;
  public B: number;
  public C: number;

  public constructor(inA: number, inB: number, inC: number) {
    this.A = inA;
    this.B = inB;
    this.C = inC;
  }
}

File: java/base/src/main/java/vsdk/toolkit/common/linealAlgebra/Jacobian.java
package vsdk.toolkit.common.linealAlgebra;

/**
Jacobian for a quadrilateral patch is J(u, v) = A + B.u + C.v
*/

public class Jacobian {
    public float A;
    public float B;
    public float C;

    public Jacobian(float inA, float inB, float inC) {
        A = inA;
        B = inB;
        C = inC;
    }
}
